### ISSUE TITLE
WebStorm integration

### ISSUE THREAD

**Issue by dyaroman** [ventojs/vento#75]
Hi. Are you planning to add WebStorm integration?

**Comment by oscarotero** [comment]
I'd love that, but I don't use WebStorm and don't know how to integrate it.

If anyone want to work on that, I'll try to help as much as I can.

**Comment by illyrius666** [comment]
yeah i wish i was advanced enough to help with that, since i need this integration too

**Comment by illyrius666** [comment]
> I'd love that, but I don't use WebStorm and don't know how to integrate it.
> 
> If anyone want to work on that, I'll try to help as much as I can.

i found quite some nice documentation on how to integrate it, and jetbrains itself also provides templates. i have created a personal project for it, but i think it would be best to add that to this organization with me as a inside collaborator for this specific project?
i have done some setup work, and iam trying to get an initial thing running, but i would need to know what exactly the syntax i need to inject is and i would need for you to adjust the project details to match that of your organization.

**Comment by oscarotero** [comment]
Hey @illyrius666, thanks for pushing this!

>  i think it would be best to add that to this organization with me as an inside collaborator for this specific project

Absolutely. I guess [this is the repository](https://github.com/illyrius666/WebStorm-Vento), right? I've sent you a invitation to be member of this organization. Let me know if you need something else.

> i would need to know what exactly the syntax i need to inject

You can take a look to the [VScode extension](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json) but basically it uses the HTML syntax as base with the following changes:

- Everything between `{{>` and `}}` is pure [JavaScript](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L34-L44).
- Everything between `{{` and `}}` is Vento code, which is also JavaScript but [with some special keywords](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L60).
- Everything between `{{#` and`#}}` is [commented code](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L15-L17).
- And it should have support [for front matters](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L63).

**Comment by illyrius666** [comment]
@oscarotero thank you for inviting me!

> Absolutely. I guess [this is the repository](https://github.com/illyrius666/WebStorm-Vento), right? I've sent you a invitation to be member of this organization. Let me know if you need something else.

I have accepted the invite and i started moving the repo to ventojs, i named it webstorm-vento all in lowercase to match the other repos.

> You can take a look to the [VScode extension](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json) but basically it uses the HTML syntax as base with the following changes:
> 
>     Everything between {{> and }} is pure [JavaScript](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L34-L44).
>     Everything between {{ and }} is Vento code, which is also JavaScript but [with some special keywords](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L60).
>     Everything between {{# and#}} is [commented code](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L15-L17).
>     And it should have support [for front matters](https://github.com/ventojs/vscode-vento/blob/main/syntaxes/vento.tmLanguage.json#L63).

I will take a look at it later after iam done with work (after 17:30 UTC+1).

**Comment by oscarotero** [comment]
🎉 Webstorm integration is work in progress here https://github.com/ventojs/webstorm-vento/issues/4

**Comment by USLTD** [comment]
Why is [it](https://github.com/ventojs/webstorm-vento) archived?

**Comment by oscarotero** [comment]
Thanks @illyrius666 for your work on this repo. Can I unarchive the repo? So people can still make PR and maybe someone even wants maintain it.
